packages/wallets/src/components/ModalProvider/ModalProvider.tsx

```diff
@@ -19,8 +19,9 @@ type TModalContext = {
 };
 
 type TModalOptions = {
-    defaultRootId?: 'wallets_modal_responsive_root' | 'wallets_modal_root';
+    defaultRootId?: 'wallets_modal_root' | 'wallets_modal_show_header_root';
```

Comment:
I'm keeping the `defaultRootId` option in case if there are cases where we need to always mount on `wallets_modal_root` like some modals in Cashier